run/navigation/demo_log_fake_navigation.cpp

```diff
@@ -0,0 +1,103 @@
+
+/*
+ * Author: Lukas Schaefer
+ * Co-Author: Justus Rudolph
+ * Organisation: HYPED
+ * Date: 01/04/19, minor edits 20/02/20
+ * Description: Demo for Multiple IMU Navigation with fake data
+ *
+ *    Copyright 2018 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "data/data.hpp"
+#include "navigation/main_log.hpp"
+#include "sensors/imu_manager.hpp"
+#include "sensors/main.hpp"
+#include "sensors/fake_gpio_counter.hpp"
+#include "utils/concurrent/thread.hpp"
+#include "utils/system.hpp"
+#include "utils/logger.hpp"
+
+using hyped::data::Data;
+using hyped::data::State;
+using hyped::data::StateMachine;
+using hyped::data::ModuleStatus;
+using hyped::data::Sensors;
+using hyped::navigation::MainLog;
+using hyped::utils::concurrent::Thread;
+using hyped::utils::System;
+using hyped::utils::Logger;
+
+int main(int argc, char* argv[])
+{
+  System::parseArgs(argc, argv);
+  System &sys = System::getSystem();
+  static Data& data = Data::getInstance();
+  Logger* log_nav = new Logger(sys.verbose_nav, sys.debug_nav);
+
+  // Keyence must be disabled
+  sys.official_run = 0;
```

Comment:
```suggestion
  sys.official_run = false;
```
It's a `bool`.
(Also, not sure if `System` was designed to be used like this but I guess it's ok in a demo file if it works.)

Reply:
Which is an `int`
Yes I am aware, that is bad practice for `system`, however as you say, it's a demo, so it's not that important.